### ISSUE TITLE
Importing Plugin - Ionic Vue

### ISSUE THREAD

**Issue by frontendjordan** [philmerrell/capacitor-firebase-analytics#10]
Hello,

I am trying to log events with the plugin but having no luck. I've built the app with @ionic/vue.

In a Home.vue single file component, I have: 

import { Plugins, DeviceInfo } from '@capacitor/core';
import { CapacitorFirebaseAnalytics } from 'capacitor-firebase-analytics';

and down in a method, I want to:

Plugins.CapacitorFirebaseAnalytics.logEvent({ name: 'school_select',  parameters: { school: this.school }});

I build and copy to Xcode and try and run the app and it crashes. Any help for a total beginner to mobile development?

**Comment by nikmartin** [comment]
is this.school a string?

**Comment by jordanmarshall12** [comment]
Good call....this.school is the whole object. I'm guessing that I need to send a string.

**Comment by frontendjordan** [comment]
Good call....this.school is the whole object. I'm guessing that I need to send a string.

**Comment by nikmartin** [comment]
yes or at least simple types like string, bool, number

**Comment by jordanmarshall12** [comment]
Still not having any luck getting this to work...

Do my imports look right to you? Instead of Plugins.CapacitorFirebaseAnalytics.logEvent I should just be doing CapacitorFirebaseAnalytics.logEvent?

**Comment by MikeCraig418** [comment]
I had a similar issue and was able to resolve it by running `npx cap update` after I installed the plugin and prior to building on my device.

I'm not using Ionic, but I am using VueJS + Capacitor.
